src/settings-ui/Settings.UI/Views/PowerPreviewPage.xaml

```diff
@@ -99,5 +99,9 @@
         <controls:SettingsPageControl.PrimaryLinks>
             <controls:PageLink x:Uid="LearnMore_PowerPreview" Link="https://aka.ms/PowerToysOverview_FileExplorerAddOns"/>
         </controls:SettingsPageControl.PrimaryLinks>
+        <controls:SettingsPageControl.SecondaryLinks>
+            <controls:PageLink Text="Aaron Junker work on developer file preview" Link="https://blog.aaron-junker.ch" />
```

Comment:
I think this is incorrect grammar. I think it has to be `Aaron Junker's` or `Aaron Junker works`.

And the part `work on` should be localized I think.

Reply:
No other attribution is localized.